GoogleSignIn/Sources/Public/GoogleSignIn/GIDSignIn.h

```diff
@@ -86,8 +86,9 @@ typedef NS_ERROR_ENUM(kGIDSignInErrorDomain, GIDSignInErrorCode) {
 /// @return `YES` if there is a previous user sign-in saved in keychain.
 - (BOOL)hasPreviousSignIn;
 
-/// Attempts to restore a previous user sign-in without interaction. Uses cached user and doesn't refresh
-/// tokens if the tokens haven't expired (<1 hour).
+/// Attempts to restore a previous user sign-in without interaction. 
+///
+/// Uses cached user and doesn't refresh tokens if the tokens haven't expired (<1 hour).
```

Comment:
I think this should explicitly state that the cache is local (on device). I also think the double negative after the "and" could be more clear.

`Restores user from the local cache and refreshes tokens if they have expired (<1 hour).`